_data/people.yml

```diff
@@ -202,18 +202,15 @@
       icon: github-alt
       name: Github
 
-- name: You
```

Comment:
Hi @pichman2 : It would be best to leave the "You" section rather than deleting it. Please return this to the bottom of the file:

```

- name: You
  pic: /img/people/you.png
  dates: now
  status: PI
  role: Group Member
  src: /positions
  url: /positions
  description: 'Ideal ARFC group members seek to advance nuclear energy challenges through advanced computing, physics-informed modeling, and sophisticated simulation'
  social:
    - url: mailto:katyhuff@gmail.com
      icon: envelope-o
      name: Email
    - url: https://www.google.com/maps/d/edit?mid=zwJ5Sc8HIDN4.k3fYHsg-biWE&usp=sharing
      icon: map-marker
      name: 118 Talbot Lab
```